github_actions/spec/dependabot/github_actions/update_checker_spec.rb

```diff
@@ -138,39 +138,50 @@
         it { is_expected.to be_falsey }
       end
 
-      context "that is a git commit SHA pointing to the tip of a branch" do
+      context "that is a git commit SHA pointing to the tip of a branch not named like a version" do
         let(:upload_pack_fixture) { "setup-node" }
-        let(:reference) { "1c24df3" }
+        let(:tip_of_master) { "d963e800e3592dd31d6c76252092562d0bc7a3ba" }
+        let(:reference) { tip_of_master }
 
-        let(:repo_url) { "https://api.github.com/repos/actions/setup-node" }
-        let(:comparison_url) { repo_url + "/compare/v1.1...1c24df3" }
-        before do
-          stub_request(:get, comparison_url).
-            to_return(
-              status: 200,
-              body: comparison_response,
-              headers: { "Content-Type" => "application/json" }
-            )
+        it { is_expected.to be_falsey }
+      end
+
+      context "that is a git commit SHA pointing to the tip of a branch named like a version" do
+        let(:upload_pack_fixture) { "run-vcpkg" }
+
+        context "and there's a branch named like a higher version" do
+          let(:tip_of_v6) { "205a4bde2b6ddf941a102fb50320ea1aa9338233" }
```

Comment:
Thanks, the addition of these `tip_of_vN` vars make the tests so much more readable!